app/logic/api_utils.R

```diff
@@ -42,15 +44,19 @@ get_api_url <- function(
 
 #' Function to get a list of all apps belonging to the token
 #'
-#' @param app_mode_filter Character. The filter for app_mode in the API
-#' response. Default is "shiny".
+#' @param app_mode_filter Character list. The filter for app_mode in the API
+#' response. Default is list("shiny", "python-shiny", "quarto-shiny").
 #' @param endpoint Character. Default is "content"
+#' @param app_role Character. Read from the config.yml file. The possible value
```

Comment:
Please add an entry to the README, so it is easy to configure the app without the need to check the code.